eg/Classes/Log.py

```diff
@@ -34,11 +36,59 @@
 
 oldStdOut = codecs.lookup("ascii").streamwriter(_oldStdOut, 'backslashreplace')
 oldStdErr = codecs.lookup("ascii").streamwriter(_oldStdErr, 'backslashreplace')
+debug_logs = {}
 
 INFO_ICON = eg.Icons.INFO_ICON
 ERROR_ICON = eg.Icons.ERROR_ICON
 NOTICE_ICON = eg.Icons.NOTICE_ICON
 
+STARTUP_DATA = [
+    '-' * 40,
+    '        %s started' % eg.APP_NAME,
+    strftime('      %d/%m/%Y - %H:%M:%S'),
+    '-' * 40,
+    '%s Version: %s' % (
+        eg.APP_NAME,
+        eg.Version.string
+    ),
+    'Windows Version: %s' % ' '.join(
+        list(platform.win32_ver())
+    ),
+    'Computer Name: %s' % platform.node(),
+    'Processor: %s' % platform.processor(),
+    'Python: %s %s' % (
+        platform.python_version(),
+        platform.python_implementation()
+    ),
+    'Python Compiler: %s' % platform.python_compiler(),
+    'Python Build Date: %s' % (
+        platform.python_build()[1],
```

Comment:
I'm missing the info that it's stackless python, so i would just use `sys.version`. It returns `'2.7.12 Stackless 3.1b3 060516 (default, Oct 31 2016, 18:42:08) [MSC v.1500 32 bit (Intel)]'
`